### PR TITLE
Upgraded some modules

### PR DESCRIPTION
**Pull request by vraravam**
### Checklist:

- [x] My pull request is properly named
- [x] The changes respect the code style of the project (`$ npm run lint`)
- [x] I tested/previewed my changes locally

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -213,6 +213,6 @@
     }
   },
   "browserslist": [
-    "Chrome 69"
+    "defaults"
```
Comment (kris7t) [inline package.json:216]: Couldn't we just have Chrome 91 here now?

Comment (vraravam) [reply package.json:216]: According this [this browserslist README](https://github.com/browserslist/browserslist#best-practices) - the `defaults` will cover all the ones that we need. So, does it matter that we need to manually keep the `Chrome` version up-to-date all the time? Or will the `defaults` mean one less thing to keep updated from time-to-time?

Comment (kris7t) [reply package.json:216]: Maybe `last 2 Chrome versions` then? I think `defaults` also transpiles for some other browsers, which probably increases the script size a bit (although maybe that doesn't matter at all).

Comment (vraravam) [reply package.json]: That's a good idea - done in the latest commit.
Have also added a call in the nightly process to update the browser db.

**Review by kris7t — APPROVED** [review]

**Pull request merged**